### DISCUSSION TITLE
Memory usage in .net8

### DISCUSSION THREAD

**Discussion by FaraSys** [dotnet/dotnet-docker#5216] [Q&A]
Hi all.
Our developers are migrating applications from .net6 to .net8 and pods are starting to consume a lot more memory.
Have you encountered this kind of behavior?
Image: alpine 3.18 .net6 pod uses ~45MB of memory.
Image: alpine 3.18 .net8 pod uses ~115MB of memory.
Image: alpine 3.19 .net8 pod uses ~115MB of memory.

**Comment by lbussell** [comment]
[Triage] You could try and use [dotnet-monitor](https://github.com/dotnet/dotnet-docker/blob/main/README.monitor.md) in your pod to collect a dump or a GC dump that you can use to analyze your application's memory usage.
